tests/components/automation/test_event.py

```diff
@@ -7,10 +7,11 @@
 from tests.common import get_test_home_assistant
 
 
+# pylint: disable=invalid-name
 class TestAutomationEvent(unittest.TestCase):
     """Test the event automation."""
 
-    def setUp(self):  # pylint: disable=invalid-name
+    def setUp(self):
```

Comment:
To do that you will need to remove inheritance on `TestCase` and migrate from `self.assert` to `assert`